docs/networking.md

```diff
@@ -0,0 +1,170 @@
+# Networking within the AWS Secure Environment Accelerator (ASEA)
+Last updated: **January 5, 2023**
+
+## Introduction
+This document breaks down the centralized networking of the AWS Secure Environment Accelerator (ASEA), and best practices for networking on the platform. It aims to make the ASEA's networking easy to grasp, helping anyone working on the platform (and within the guardrails of the BC Gov ASEA) understand how our networking is set up. Good networking is like the backbone of cloud technology. It connects everything, keeps things safe, and helps things run smoothly. Understanding it is crucial for making the most of cloud tools.
```

Comment:
```suggestion
This document simplifies the centralized networking of the AWS Secure Environment Accelerator (ASEA) and provides best practices for networking on the platform. Our goal is to make ASEA's networking easy to understand for anyone working on the platform within the guardrails of the BC Gov ASEA. Networking plays a crucial role as the backbone of cloud technology, connecting everything, ensuring safety, and facilitating smooth operations. Understanding it is essential for maximizing the benefits of cloud tools.
```

Reply:
Thank you for pointing this out @MaxWardle! I forgot to actually paste my correction on the review, please see below: